core/modules/replica/DatabaseMySQL.cc

```diff
@@ -423,13 +423,13 @@ void Connection::exportField(ProtocolResponseSqlField* ptr,
         throw out_of_range(context + " error: index is out of range");
     }
     auto&& field = _fields[idx];
-    ptr->set_name(      field.name,      field.name_length);
-    ptr->set_org_name(  field.org_name,  field.org_name_length);
-    ptr->set_table(     field.table,     field.table_length);
-    ptr->set_org_table( field.org_table, field.org_table_length);
-    ptr->set_db(        field.db,        field.db_length);
-    ptr->set_catalog(   field.catalog,   field.catalog_length);
-    ptr->set_def(       field.def,       field.def_length);
+    ptr->set_name(      field.name/*,      field.name_length*/);
+    ptr->set_org_name(  field.org_name/*,  field.org_name_length*/);
+    ptr->set_table(     field.table/*,     field.table_length*/);
+    ptr->set_org_table( field.org_table/*, field.org_table_length*/);
+    ptr->set_db(        field.db/*,        field.db_length*/);
+    ptr->set_catalog(   field.catalog/*,   field.catalog_length*/);
+    ptr->set_def(       field.def/*,       field.def_length*/);
```

Comment:
it looks like this block was changed in the previous commit, and then changed back here. You could just `fixup` this commit into the previous one and not have this change. Or: why is there commented code here, can it go away?

Reply:
I'm sorry, the commit structure is not perfect! But I'm working on it (making my commits as perfect as yours). The commented blocks were removed later.